### PR TITLE
Performed API call to get job status using Redux.

### PR DESCRIPTION
**Pull request by ettienekorayyi**
## Rationale
Created the getStatus action creator that gets called by the Redux Thunk middleware to perform GET request the '/lists' endpoint.

This is for testing the git workflow

## Advice for Reviewers & Testing Notes

- Explain how your change should be reviewed, and where reviewers should direct their attention.
- Explain (where relevant) how your change can be tested.

## Screenshots:
- Add a screenshot or two of your changes, where applicable.
image

image

## Task Name
Job-Management-Create-Job

- Issue Title

## Linting Checklist
- [ ] No commented code
- [ ] Code Formatted nicely (Prettier)
- [ ] PR your own code before you assign reviewers

### CODE REVIEW

**Review by giftakari — CHANGES_REQUESTED** [review]
Hi Stephen,

Everything looks okay to me. 

Can you please create a .evn file and not share the token publicly on GitHub ?

**Comment by ettienekorayyi** [comment]
> Hi Stephen,
> 
> Everything looks okay to me.
> 
> Can you please create a .evn file and not share the token publicly on GitHub ?

Hi Gift,

I will, thanks

**Comment by ettienekorayyi** [comment]
Hi Gift,

I will, thanks

File: src/App.js [hunk 1]
```diff
@@ -3,20 +3,26 @@ import SignUp from "./components/Forms/SignUp";
...
+      <div className='App'>
+        <Header />
+        <RightSideBar />
+        <Switch>
+          <Route exact={true} path='/' component={NewInventory} />
+          <Route exact path='/login' component={TradieLogin} />
+          <Route exact path='/create/job' component={CreateJob} />
+        </Switch>
+      </div>
+    </Provider>
```
Comment (kimchirice) [inline src/App.js:25]: LGTM. but having a question about the where to setup ```Redux```.

should we move ```Provider``` and ```store``` to src/index.js??

File: src/api/taskstechApi.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+const authToken = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoyMSwiZXhwIjoxNjI2NTAyMjExfQ.i9E85u3U72x_amkKriiBL5HArbMX5k4l_rFgcXiq09g';
```
Comment (kimchirice) [inline src/api/taskstechApi.js:2]: I guess the auth token would be retrived from local storage once the ```Login``` component is in place. 

but like @giftakari mentioned, that it's best to eliminate the chance to expose security details to the public.

**Review by kimchirice — CHANGES_REQUESTED** [review]

File: src/api/taskstechApi.js (hunk 2 above)
Comment (ettienekorayyi) [reply src/api/taskstechApi.js]: Thanks for the feedback, I have created the .env file.

File: src/App.js (hunk 1 above)
Comment (ettienekorayyi) [reply src/App.js:24]: Hi @kimchirice , yes I will move the Redux setup to the index.js,

Comment (ettienekorayyi) [reply src/App.js]: Hi @kimchirice I have moved the Redux setup to the index.js

**Review by kimchirice — APPROVED** [review]
LGTM.

btw, I add all the team members to the reviewers, at least go thru the changes whenever you are up for it. 

for how to review PR, check the [our project Wiki pages of How to review new PR from your local repo??](https://github.com/codesydney/taskstech/wiki#howto)

**Review by Fumicode01 — APPROVED** [review]

**Review by Githubzoe — APPROVED** [review]

**Pull request merged**